pkgvars

```diff
@@ -1,8 +1,7 @@
 rpm_pkgname=pg-auto-failover
 deb_pkgname=auto-failover
+pkgname=pg-auto-failover
 hubproj=pg_auto_failover
 pkgdesc='Postgres extension for automated failover and high-availability'
-pkglatest=1.6.2-1
-releasepg=10,11,12,13,14
-nightlypg=10,11,12,13,14
```

Comment:
AFAIK we only keep the PG versions for Citus in the matrix https://github.com/citusdata/tools/blob/develop/packaging_automation/tests/files/postgres-matrix.yml

If we remove these lines, how will we decide on which PG versions that this extension will support?

Reply:
From postgres-matrix  again. There is no use for these parameters now. I keeped them until remove all pipelines from Travis

Reply:
We had an offline discussion and as it turns out, I should take a look at https://github.com/citusdata/packaging/blob/all-pgautofailover/postgres-matrix.yml instead of https://github.com/citusdata/tools/blob/develop/packaging_automation/tests/files/postgres-matrix.yml